src/components/ExportCampaignDataSnackbar.tsx

```diff
@@ -0,0 +1,30 @@
+import Snackbar from "@material-ui/core/Snackbar";
+import Alert from "@material-ui/lab/Alert";
+import React from "react";
+
+interface Props {
```

Comment:
required: export props, and name `ExportCampaignDataSnackbarProps`, more details in https://github.com/politics-rewired/Spoke/blob/main/conventions.md